src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json

```diff
@@ -29,6 +29,8 @@
     "sulu_admin.error_required": "Dieses Feld muss ausgefüllt werden",
     "sulu_admin.error_minlength": "Die minimale Länge dieses Feldes wurde unterschritten",
     "sulu_admin.error_minitems": "Dieses Feld hat zu wenig Zuweisungen",
+    "sulu_admin.error_maxitems": "Dieses Feld hat zu viele Zuweisungen",
+    "sulu_admin.error_type": "Ein Fehler ist aufgetreten",
```

Comment:
This error should not occurr and is likely a bug, therefore I decided to use a generic error message